workspace/Foo/app.json

```diff
@@ -126,6 +127,9 @@
         "path": "${toolkit.name}/resources"
     }, {
         "path": "${build.id}/resources"
+    },{
+        "path": "_resources",
+        "output": "_resources"
```

Comment:
Instruct Sencha Cmd to copy whatever is inside _resources ("path") to the pool named _resources. (See change above, on "output")